tests/gold_tests/autest-site/microserver.test.ext

```diff
@@ -114,15 +129,16 @@ def MakeOriginServer(obj, name, port=False, ip=False, delay=False, ssl=False, op
         ip = '127.0.0.1'
     if (delay == False):
         delay = 0
-    command = "python3 {0} --data-dir {1} --port {2} --ip_address {3} --delay {4} -m test --ssl {5}".format(
-        server_path, data_dir, port, ip, delay, ssl)
+    command = "python3 {0} --data-dir {1} --port {2} --ip_address {3} --delay {4} -m test --ssl {5} --lookup_key {6}".format(
```

Comment:
Not `lookup_key` and `data-dir`. Let's not have mix and match with `-` and `_` in the options. Dang, I didn't even notice `ip_address`.